lib/js/styles/Colors/Colors.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<div class="sb__presentation">
+		<div class="sb__row">
+			<ol class="sb__list">
+				<li v-for="(colors, colorCategory) in rawColors" :key="colorCategory" class="sb__legend">
+					{{ colorCategory }}
+				</li>
+			</ol>
+		</div>
+		<div v-for="(colors, colorCategory) in rawColors" :key="colorCategory" class="sb__row">
+			<h2>{{ colorCategory }}</h2>
+			<div v-for="color in colors" :key="color.id" class='sb__table'>
+				<div class="sb__tableId">{{ color.label }}</div>
+				<div class="sb__tableValue">{{ color.value }}</div>
+				<div class="sb__tableColor">
+					<span class="sb__color" :style="{ background: color.value }"></span>
+				</div>
+			</div>
+		</div>
+	</div>
+</template>
+
+<style lang="scss">
+.sb {
+	&__presentation {
+		width: 80%;
+		margin: 0 auto;
+
+		h2 {
+			margin-top: 0;
+		}
+	}
+
+	&__row {
+		border: 1px solid #f3f2f2;
+		padding: 20px  20px;
+		margin-bottom: 20px;
```

Comment:
we try to keep alphabetical order of style attributes

Reply:
Roger that.